src/datumaro/experimental/fields.py

```diff
@@ -21,6 +21,13 @@
 T = TypeVar("T")
 
 
+@dataclass
+class Label:
+    """Minimal Label annotation class."""
+
+    label: Any  # Can be int, str, or other types for flexibility
```

Comment:
This class is not needed. When creating a sample, the user can specify whatever type they want. E.g.
```
class MySample(Sample):
    label: int = label_field()
```

In this example, we don’t use a label class. The data is directly converted to int.